src/Routes/ImageManagerDetail/ImageDetailActions.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Text,
+  Button,
+  SplitItem,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+} from '@patternfly/react-core';
+
+const ImageActions = ({ imageData, openUpdateWizard }) => {
+  console.log(imageData);
+  const [isOpen, setIsOpen] = useState(false);
+
+  const dropdownItems = [
+    <DropdownItem
+      listItemClassName="remove-list-style pf-u-mr-lg"
+      href={imageData?.Installer?.ImageBuildISOURL}
+      key="link"
+    >
+      <Text className="force-text-black">Download</Text>
+    </DropdownItem>,
+  ];
+
+  const handleToggle = (isOpen) => setIsOpen(isOpen);
+
+  const handleSelect = () => {
+    setIsOpen((prevState) => !prevState);
+    onFocus();
+  };
+
+  const onFocus = () => {
+    const element = document.getElementById('toggle-id-6');
```

Comment:
what happens if you add a new element above it. Then it won't be `toggle-id-6` it could be `toggle-id-10`.
Maybe you can use `useRef` here instead. However, if the element isn't being focused on select, this should be a bug in patternfly core. So make sure to open an issue on their repository here: https://github.com/patternfly/patternfly-react/issues

Reply:
I removed onFocus, it wasn't necessary.